src/test/java/task_modification_use_case/TaskModificationPresenterTest.java

```diff
@@ -0,0 +1,35 @@
+package task_modification_use_case;
+import org.junit.jupiter.api.Test;
+
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskModificationPresenterTest {
+    //task and event instances (and maps) to be used across tests.
+    private final static TaskModificationResponseModel task = new TaskModificationResponseModel(
+            "Sample Task", 0
+    );
+    private final static String error = "ERROR!";
+
+    @Test
+    public void testPrepareSuccessView(){
+        TaskModificationOutputBoundary outputBoundary = new TaskModificationPresenter();
+        TaskModificationResponseModel responseModel = outputBoundary.prepareSuccessView(
+                task
+        );
+        assertEquals(task.getDayId(), responseModel.getDayId());
+        assertEquals(task.getTitle(), responseModel.getTitle());
+    }
+
+    @Test
+    public void testPrepareFailView(){
```

Comment:
Would appreciate clarification on what you believe the PresenterTest file should be tesing?

Reply:
I believe what I meant was that because the presenter gets passed in the OutputData, it needs to check that said OutputData was modified in whatever way was necessary rather than check if it created an OutputData

Reply:
I have addressed this in the latest commit.